tests/package-show-custom-coverage-test.js

```diff
@@ -37,14 +57,60 @@ describeApplication('PackageShowCustomCoverage', () => {
     it('displays the custom coverage section', () => {
       expect(PackageShowPage.customCoverage).to.equal('7/16/1969 - 12/19/1972');
     });
+
+    // TODO: why use `it.still` here instead of just `it`?
+    // if `it.still` is needed here, why not for the other assertions
+    // in this `describe` block?
```

Comment:
dunno if we need this TODO anymore (which i added in the first place, sorry about that)